model_analyzer/config/input/config_field.py

```diff
@@ -174,6 +179,8 @@ def set_default_value(self, default_value):
 
         self._default_value = default_value
 
+        self._set_by_user = False
```

Comment:
is set by user used anywhere?

Reply:
Good catch, this should be set_by_config